astrocut/make_cube.py

```diff
@@ -440,7 +440,7 @@ def _configure_cube(self, file_list, **extra_keywords):
         # Determining cube shape: 
         # If it's a new cube, the shape is (nRows, nCols, nImages, 2)
```

Comment:
this comment should be updated, maybe include something about the reason it is 1 instead of 2